en/mkdocs.yml

```diff
@@ -786,289 +786,289 @@ plugins:
     - markdownextradata: {}
     - redirects:
           redirect_maps:
-              'learn/tutorials.md': 'https://is.docs.wso2.com/en/6.0.0/'
-              'learn/adding-users-and-roles.md': 'https://is.docs.wso2.com/en/6.0.0/'
-              'learn/managing-user-accounts.md': 'https://is.docs.wso2.com/en/6.0.0/'
-              'learn/user-registration.md': 'https://is.docs.wso2.com/en/6.0.0/'
-              'learn/user-registration-with-password-entry.md': 'https://is.docs.wso2.com/en/6.0.0/'
-              'learn/user-registration-with-email-verification.md': 'https://is.docs.wso2.com/en/6.0.0/'
-              'learn/self-registration.md': 'https://is.docs.wso2.com/en/6.0.0/'
-              'learn/configuring-password-policies.md': 'https://is.docs.wso2.com/en/6.0.0/'
-              'learn/configuring-password-patterns.md': 'https://is.docs.wso2.com/en/6.0.0/'
-              'learn/validating-password-history.md': 'https://is.docs.wso2.com/en/6.0.0/'
-              'learn/configuring-password-reset.md': 'https://is.docs.wso2.com/en/6.0.0/'
-              'learn/configuring-password-reset-with-email.md': 'https://is.docs.wso2.com/en/6.0.0/'
-              'learn/configuring-password-reset-with-challenge.md': 'https://is.docs.wso2.com/en/6.0.0/'
-              'learn/configuring-user-name-recovery.md': 'https://is.docs.wso2.com/en/6.0.0/'
-              'learn/configuring-account-locking.md': 'https://is.docs.wso2.com/en/6.0.0/'
-              'learn/configuring-admin-initiated-account-locking.md': 'https://is.docs.wso2.com/en/6.0.0/'
-              'learn/configuring-account-locking-due-to-failed-login-attempts.md': 'https://is.docs.wso2.com/en/6.0.0/'
-              'learn/configuring-account-disabling.md': 'https://is.docs.wso2.com/en/6.0.0/'
-              'learn/pending-account-status.md': 'https://is.docs.wso2.com/en/6.0.0/'
-              'learn/configuring-single-sign-on-saml.md': 'https://is.docs.wso2.com/en/6.0.0/'
-              'learn/configuring-single-sign-on-oidc.md': 'https://is.docs.wso2.com/en/6.0.0/'
+              'learn/tutorials.md': 'https://is.docs.wso2.com/en/6.0.0/guides/before-you-start/'
+              'learn/adding-users-and-roles.md': 'https://https://is.docs.wso2.com/en/6.0.0/guides/identity-lifecycles/user-management/'
```

Comment:
this one is wrong isn't it
`https://` duplicated

Reply:
Yeas, Will update it thankss